tests/lib/rules/spaced-comment.js

```diff
@@ -270,8 +273,9 @@ ruleTester.run("spaced-comment", rule, {
         },
         {
             code: "//*********************-\n// Comment Block 3\n//***********************",
+            output: "//* ********************-\n// Comment Block 3\n//***********************",
```

Comment:
Sorry, I don't think this is right result.
`exceptions` option allows repeating until a line break.

The correct output is: `//*********************\n// Comment Block 3\n//***********************`